.studiorc

```diff
@@ -273,6 +273,35 @@ origin() {
   fi
 }
 
+test-builder() {
+    hab sup status
+    if [ $? -eq 0 ]; then
```

Comment:
Is this a reliable way to check if the sup is running? I see on line 91 of this file we check the text output. If it is a reliable way, both ways of checking for sup aliveness should match

Reply:
This is a reliable way. It checks the exit code of running the previous command. Since we're checking up/down-ness from an expected unclean state the intention of these two checks are slightly different. The up check on line 91 is intended for a clean state and validates both that the sup is up AND that there are services loaded. In the case of 278, we are only validating that sup is up.